test-harness/scenarios/help-no-document.scenario

```diff
@@ -2,13 +2,15 @@
 Errors when no document is provided
 ====command====
 lint
+====env====
+isTTY=true
```

Comment:
Could we get a test harness test showing `cat somefile.yaml | spectral --something` to show that its working as expected?